### ISSUE TITLE
"Interview my parents" feature

### ISSUE THREAD

**Issue by tedbarnett** [tedbarnett/Boswell#149]
The most important people to interview are senior citizens (60+). We need to make it as simple as possible for my 86-year-old mom to use. We also need easy sharing of the app to get it onto my mom's iPhone. So...

We design a hidden "Silver" Mode for Boswell: 
- same features but simplified to the extreme
- Only runs in Boswell mode (no ChatGPT etc.)
- Has at least a few selected photos to ask about (see below)
- Has name and birthdate already set (see below)
- can automatically share new interviews with the "share source" user (ie When mom clicks the share button, I am the default recipient?)

Allow the son or daughter ("share source") to choose new menu item "Interview Parent"
Selecting that brings up a dialog asking for parent's name, birthdate. Also asks for 5-10 new photos to be asked about. So I can "pre-configure" Boswell for my Mom. 

I can then share this new "Silver" version just as I can share the PDF now. I send it to my Mom's iMessage for example. Message says 
- 'Ted asked you help fill in some blanks on our family history and photos. Click here to install "Boswell", which will interview you for us. {install link}"

Clicking that link opens Boswell in the App Store and pre-loads the data (and photos?) already selected by the son/daughter. 

Future feature: son can send more photos to mom and they are auto-added to mom's Boswell

**Comment by tedbarnett** [comment]
Let's make Silver mode its own hidden menu item. A checkbox in My Profile labeled "Silver mode" with a hover-over description: "Simplifies Boswell interface". 
Default is unchecked. If checked, switches to Silver mode.

**Comment by bhautikpatoliya004** [comment]
@tedbarnett  Yes, it's possible to share PDFs for some contacts only in iMessage and Email Using [MFMessageComposeViewController](MFMessageComposeViewController) and [MFMailComposeViewController](https://developer.apple.com/documentation/messageui/mfmailcomposeviewcontroller)

**Comment by bhautikpatoliya004** [comment]
It's not possible to share "pre-configure" app. We can share apps using  Universal link with some data but here we need our server setup so without a server it's not possible to share "pre-configure" app.

**Comment by tedbarnett** [comment]
I was assuming the app itself would not be shared: just a link. That link would then "configure" the app when it is installed so that it ends up in Silver mode with name and birthdate. 

We could figure out how to send photos over later. Right?

**Comment by tedbarnett** [comment]
For now we just need the "Silver" mode checkbox to demo to folks.
